otter/util/fp.py

```diff
@@ -122,3 +122,8 @@ def set_in(mapping, keys, new_value):
     else:
         child = mapping.get(keys[0], pmap())
         return mapping.set(keys[0], set_in(child, keys[1:], new_value))
+
+
+def raise_(e):
```

Comment:
Reimplementing this by accident is apparently not a new problem though: https://github.com/python-effect/effect/issues/72